src/SDKs/WebSites/Management.Websites/Microsoft.Azure.Management.Websites.csproj

```diff
@@ -19,5 +19,8 @@
 
 	<!-- Please do not move/edit code below this line -->
 	<Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
+	<ItemGroup>
+	  <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.15" />
```

Comment:
Please remove this

Reply:
@dsgouda had to add this due to a bug in .NET SDK with provisioning state polling code . see the email below for details
@shahabhijeet as FYI as well.
----------------------------------------------------------------------------------------------------------


Can you update to the latest version of Microsoft.Rest.ClientRuntime.Azure and that should do the fix for that particular error.

Abhijeet

From: Nicholas King 
Sent: Wednesday, August 22, 2018 3:43 PM
To: Abhijeet Shah 
Hi Abhijeet,

Sisira and I are working on updating the Web Apps SDK right now, and we’ve gotten stuck recording some of our tests. I dug through the code and noticed you’ve been working on the provisioning state polling code. Hopefully you’ll be able to help us out.

We have a test that creates an AppServicePlan (a.k.a. ServerFarm internally), then updates the AppServicePlan. Both creating and updating are done with PUT requests to the same URL. When creating the AppServicePlan the response contains provisioningState: Succeeded, but when updating it, the response contains provisioningState: null. The internal SDK Client is throwing an exception “Provisioning state is missing from long running operation.”

I attached a recording of the failing test. My main question for now is, is there an issue with how our API currently works, or is the problem fixable inside the .NET SDK?

Thanks
Nick

Reply:
Thanks for confirming. We need to take a look at this and decide if we can allow a one off here or if we are ready to move our ClientRuntime dependency to the latest version for all sdks.

Reply:
@panchagnula Please revert this change, we are planning to bump the version dependency for all SDKs across the repo, will merge this PR once that is done.